task/eaas-provision-space/0.1/eaas-provision-space.yaml

```diff
@@ -59,20 +59,36 @@ spec:
         esac
 
         cat <<EOF > space_request.yaml
-        apiVersion: toolchain.dev.openshift.com/v1alpha1
-        kind: SpaceRequest
+        apiVersion: apiextensions.crossplane.io/v1
+        kind: CompositeResourceDefinition
         metadata:
-          generateName: eaas-spacerequest-
-          namespace: $NAMESPACE
+          name: xnamespaces.eaas.konflux-ci.dev
           ownerReferences:
           - apiVersion: tekton.dev/v1
             kind: $OWNER_KIND
             name: $OWNER_NAME
             uid: $OWNER_UID
+          annotations:
+            argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
```

Comment:
ArgoCD doesn't manage resources in tenant namespaces; don't include such annotations.